tests/describe_task_queue_test.go

```diff
@@ -245,14 +293,14 @@ func (s *DescribeTaskQueueSuite) validateDescribeTaskQueue(
 				Versions:               nil, // default version, in this case unversioned queue
 				TaskQueueTypes:         nil, // both types
 				ReportPollers:          true,
-				ReportTaskReachability: true,
+				ReportTaskReachability: false,
```

Comment:
I don't think this should be set to `true` here since `describe_task_queue` for reachability has already been tested within versioning_test.go. (pending to move those tests in this folder)